meeteval/io/base.py

```diff
@@ -353,31 +354,16 @@ def filenames(self):
     def to_seglst(self) -> 'SegLST':
         from meeteval.io.seglst import SegLST
         return SegLST([l.to_seglst_segment() for l in self.lines])
+    
+    def map(self, fn):
```

Comment:
Have you removed the map from seglst? If not, is it different?

Reply:
No, `segLST` uses different namings (`self.segments` vs `self.lines`) and doesn't inherit from `Base`, only from `BaseABC`. I'll keep both for now.